ert/data/_transformation.py

```diff
@@ -0,0 +1,66 @@
+import pathlib
+from pathlib import Path
+import tarfile, os, io
+import aiofiles
+from ert.serialization import get_serializer
+import stat
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from ert.data import RecordTransmitter
+
+_BIN_FOLDER = "bin"
+
+
+class FileTransformation:
+    def __init__(self, name="DEFAULT"):
+        self._name = name
```

Comment:
What's the name for really? If it is just identity, can't we use `isinstanceof`?

Reply:
Yes, I thought I might fond use for it, but didn't :)